### ISSUE TITLE
Why using F.binary_cross_entropy after softmax?

### ISSUE THREAD

**Issue by Starethics** [vandit15/Class-balanced-loss-pytorch#23]
shouldn't it be F.cross_entropy instead?
